cADFS.psm1

```diff
@@ -883,7 +883,8 @@ function AddADFSNode {
 
     Add-AdfsFarmNode -CertificateThumbprint:$CertificateThumbprint `
     -ServiceAccountCredential $serviceCredential `
-    -PrimaryComputerName $PrimaryADFSServer
+    -PrimaryComputerName $PrimaryADFSServer `
+    -OverwriteConfiguration $true
```

Comment:
Can you make this a property that can be configured on the resource itself? It should default to false in case someone does this by accident and wipes out the config they had.